etc/lower-minheartbeatfrequencyms.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/Sources/CLibMongoC/mongoc/mongoc-topology-private.h b/Sources/CLibMongoC/mongoc/mongoc-topology-private.h
+index 350b3b5..0e8ea28 100644
+--- a/Sources/CLibMongoC/mongoc/mongoc-topology-private.h
++++ b/Sources/CLibMongoC/mongoc/mongoc-topology-private.h
+@@ -28,7 +28,7 @@
+ #include "mongoc-client-session-private.h"
+ #include "mongoc-crypt-private.h"
+ 
+-#define MONGOC_TOPOLOGY_MIN_HEARTBEAT_FREQUENCY_MS 500
++#define MONGOC_TOPOLOGY_MIN_HEARTBEAT_FREQUENCY_MS 50
```

Comment:
This seems dangerous, are you enforcing this minimum elsewhere? It may speed up test runs, but in actual deployments this could result in very tight loops in e.g. failover scenarios.

Reply:
We are enforcing the minimum in Swift now, see [here](https://github.com/mongodb/mongo-swift-driver/pull/526#discussion_r489829243).

Reply:
The previous validation wasn't sufficient for limiting immediate SDAM checks; updated to allow dynamic setting of the minHeartbeatFrequencyMS via another patch in mongoc. Now we manually set it for every client with a default of 500 which can be configured as in previous patches of this PR.